script.js

```diff
@@ -1,198 +1,149 @@
+                },
         
-    //Get highest temperature forecast for today from datahub
-        var current_high_temp = daily_result.features[0].properties.timeSeries[1].dayMaxScreenTemperature;
-        current_high_temp = Math.round(current_high_temp);
-        var element = document.getElementById("currentHighTemp");
-        element.innerHTML = current_high_temp +'°C';
-
-    //Get lowest temperature forcast for today from datahub
-        var current_low_temp = daily_result.features[0].properties.timeSeries[1].nightMinScreenTemperature;
-        current_low_temp = Math.round(current_low_temp);
-        var element = document.getElementById("currentLowTemp");
-        element.innerHTML = current_low_temp +'°C';
-
-//5 Day Forecast        
-
-    //Get temperature range for Day 1 of forecast from datahub
-        var day1_high_temp = daily_result.features[0].properties.timeSeries[2].dayMaxScreenTemperature;
-        day1_high_temp = Math.round(day1_high_temp);
-        var element = document.getElementById("day1HighTemp");
-        element.innerHTML = day1_high_temp +'°C';
-
-        var day1_low_temp = daily_result.features[0].properties.timeSeries[2].nightMinScreenTemperature;
-        day1_low_temp = Math.round(day1_low_temp);
-        var element = document.getElementById("day1LowTemp");
-        element.innerHTML = day1_low_temp +'°C';
-
-    //Get significant weather code from data hub
-        var day1_weather_code = daily_result.features[0].properties.timeSeries[2].daySignificantWeatherCode;
-        day1_weather_code = day1_weather_code.toString();
-        var element = $("#day1WeatherCode");
-        element.innerHTML = day1_weather_code;
-
-    //Use weather code to append source for corresponding icon
-        element.attr('src', `images/weather-${day1_weather_code}.png`);
+                success:function(hourly_result) {
+                    console.log(hourly_result);
+                
+            const hourly = hourly_result.features[0].properties.timeSeries[2];
+                
+            //Get current temperature from datahub and append
+                let current_temp = (hourly).screenTemperature;
+                current_temp = Math.round(current_temp);
+                let element4 = $("#currentTemp");
+                element4.html(current_temp +'°C');
+            
+            //Get current weather code from datahub
+                let current_weather_code = (hourly).significantWeatherCode;
+                current_weather_code = current_weather_code.toString();
+                let element5 = $("#currentWeatherCode");
+                element5.innerHTML = current_weather_code;
+            
+            //Use weather code to append source for corresponding icon        
+                element5.attr('src', `images/weather-${current_weather_code}.png`);
+                console.log(current_weather_code);
+                },
+                })
```

Comment:
I think it'd be worth splitting this out into it's own function too `getHourlyData()` maybe